daemon/algod/api/algod.oas2.json

```diff
@@ -815,6 +840,163 @@
         }
       }
     },
+    "ApplicationStateSchema": {
+      "description": "Specifies maximums on the number of each type that may be stored.",
+      "type": "object",
+      "required": [
+        "num-uint",
+        "num-byte-slice"
+      ],
+      "properties": {
+        "num-uint": {
+          "description": "\\[nui\\] num of uints.",
+          "type": "integer",
+          "x-algorand-format": "uint64"
+        },
+        "num-byte-slice": {
+          "description": "\\[nbs\\] num of byte slices.",
+          "type": "integer",
+          "x-algorand-format": "uint64"
+        }
+      },
+      "x-go-name": "StateSchema"
+    },
+    "ApplicationLocalStates": {
+      "description": "Pair of application index and application local state",
+      "type": "object",
+      "required": [
+        "app-index",
+        "state"
+      ],
+      "properties": {
+        "app-index": {
+          "type": "integer",
+          "x-algorand-format": "uint64"
+        },
+        "state": {
+          "$ref": "#/definitions/ApplicationLocalState"
+        }
+      }
+    },
+    "ApplicationLocalState": {
+      "description": "Stores local state associated with an application.",
+      "type": "object",
+      "required": [
+        "schema",
+        "key-value"
+      ],
+      "properties": {
+        "schema": {
+          "description": "\\[hsch\\] schema.",
+          "$ref": "#/definitions/ApplicationStateSchema"
+        },
+        "key-value": {
+          "description": "\\[tkv\\] storage.",
+          "$ref": "#/definitions/ApplicationKeyValueStore"
+        }
+      },
+      "x-go-name": "AppLocalState"
+    },
+    "ApplicationKeyValueStore": {
+      "description": "Represents a key-value store for use in an application.",
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/ApplicationKeyValue"
+      },
+      "x-go-name": "TealKeyValue"
+    },
+    "ApplicationKeyValue": {
+      "description": "Represents a key-value pair in an application store.",
+      "type": "object",
+      "required": [
+        "key",
+        "value"
+      ],
+      "properties": {
+        "key": {
+          "type": "string"
+        },
+        "value": {
+          "$ref": "#/definitions/ApplicationValue"
+        }
+      }
+    },
+    "ApplicationValue": {
```

Comment:
I'm not following what this is, exactly. There's an enum format that can be applied to strings if thats what you're trying to get.

Reply:
this is one-to-one mapping for basics.TealValue